marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Good. Thanks @graimundo. @pminutillo, @wseyler, @hbfernandes, @tmcsantos, @ViswesvarSekar, could you review it then?